cnf-certification-test/identifiers/identifiers.go

```diff
@@ -473,15 +462,20 @@ func getGinkgoTestID(identifier claim.Identifier, extra string) string {
 //   http://test-network-function.com/tests-case/SuitName/TestName
 // It extracts SuitNAme and TestName
 
-func GetSuiteAndTestFromIdentifier(identifier claim.Identifier) []string {
+func GetSuiteAndTestFromIdentifier(identifier claim.Identifier) (suiteName, testName string) {
 	result := strings.Split(identifier.Url, url+"/")
 	const SPLITN = 2
 	// len 2, the baseDomain can appear only once in the url
 	// so it returns what you have previous and before basedomain
 	if len(result) != SPLITN {
-		return nil
+		panic("Invalid Identifier Url")
 	}
-	return strings.Split(result[1], "/")
+
+	result = strings.Split(result[1], "/")
+	suiteName = result[0]
+	testName = result[1]
+
+	return suiteName, testName
```

Comment:
Was this just for readability?

Reply:
I believe the linter was complaining about some requirement requiring the return values to be named. I can recheck this.